src/riak_kv_delete.erl

```diff
@@ -94,6 +97,7 @@ delete(ReqId,Bucket,Key,Options,Timeout,Client,ClientId,VClock) ->
                     ?DTRACE(?C_DELETE_INIT2, [1], [<<"reap">>]),
                     {ok, C2} = riak:local_client(),
                     AsyncTimeout = 60*1000,     % Avoid client-specified value
+                    timer:sleep(TombPause),
```

Comment:
Explain, why is 10 milliseconds a good time to resolve the issues you have?

Reply:
The delay is harmless, in the sense the client has already had a response (although it will slow down deletes queued by riak_kv_eraser).

I suspect 1ms will make it go away, there'a network hop involved in the race condition.  However, other environments might be different.  

It is configurable if it proves to be a problem, so will leave at 10ms until more data comes back from deployments.